api/src/auth.rs

```diff
@@ -119,71 +176,74 @@ impl UserDirectory {
     /// will return an error since the project does not belong to the current user
```

Comment:
Just need to update the comment two lines above this since we will now also fail if the user owns the project